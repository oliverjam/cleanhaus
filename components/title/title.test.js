import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Title from "./title";

jest.unmock("./title");

describe("Title component", () => {
  it("Renders correctly", () => {
    const title = renderer.create(<Title>Lorem Ipsum</Title>).toJSON();
    expect(title).toMatchSnapshot();
  });
});
