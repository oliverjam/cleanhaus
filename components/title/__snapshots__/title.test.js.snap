// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component Renders correctly 1`] = `
.c0 {
  font-size: 2em;
  font-weight: normal;
}

<h1
  className="c0"
>
  Lorem Ipsum
</h1>
`;
